### ISSUE TITLE
Mobile browser issues

### ISSUE THREAD

**Issue by paschaldev** [aurelia/framework#577]
I just discovered that the official site http://aurelia.io does not load on firefox on android as well as UC Browser.

It works flawlessly on chrome browser but doesn't on firefox and UC Browser. I believe the official site was written in aurelia also. Does this have to do with browser support? Does aurelia supports only chrome on android?

Lastly, please there should be a page for browser support with their versions just like how angular has it on their github homepage. Its hard to guess which browser version aurelia works on.

**Comment by paschaldev** [comment]
I'm working on an application which I intend to build with aurelia but the mobile browser support scares me..

**Comment by EisenbergEffect** [comment]
Sorry, what is UC?

**Comment by EisenbergEffect** [comment]
We don't support UC explicitly. I haven't heard of any issues with Firefox before this.

**Comment by EisenbergEffect** [comment]
I just tested Firefox on Android and the site and hub worked flawlessly.

**Comment by EisenbergEffect** [comment]
We support Evergreen Browsers. We also support older browsers, back to IE9 era, by adding additional polyfills.

**Comment by paschaldev** [comment]
This is a little bit strange. My android is 1GB ram. Firefox browser was just freshly ibstalled feom play store. This was my observation.

I used chrome, works flawlessly except for one issue, the code blocks sometimes refuse to scroll horizontally so I decided to install firefox to try.

1:14am: Launched firefox and visited other sites like google, FB. All worked.
At that same time, closed all tabs and tried visiting aurelia.io... 2seconds, it already fetched d page title "Home | Aurelia" but the progress bar was still at the middle, didn't load any page content yet...

11:17: Waited 3minutes thinking it was network issue, no changes yet. I thought d firefox had issues, so I opened another tab and visited other sites. They all opened immediately. My next thought was that aurelia server was down.

11:19: I launched chrome, visisted aurelia.. Worked flawlessly. I went back to firefox, reload page, same effect. Loads page title but page content won't load. Progress ar jist stuck at d middle.

While waiting for firefox, i could access d hub and every otger part of aurelia on chrome.

Firefox version is 4.3.

I can't really understand whats going on.
